provider/marathon.go

```diff
@@ -150,7 +151,10 @@ func taskFilter(task marathon.Task, applications *marathon.Applications) bool {
 		log.Errorf("Unable to get marathon application from task %s", task.AppID)
 		return false
 	}
-	if application.Labels["traefik.enable"] == "false" {
+	if !defaultExposeFlag && application.Labels["traefik.enable"] != "true" {
```

Comment:
This looks a little bit complicated to read. We could maybe use something like : 

```go
if !isApplicationEnabled(application) {
    // […]
    return false
}

// […]

func isApplicationEnabled(application *marathon.Application, exposedByDefault bool) bool {
    return exposedByDefault && application.Labels["traefik.enable"] != "false" || application.Labels["traefik.enable"] == "true"
}
```